### PR TITLE
Ruletest 23-8 Test JSON

### PR DESCRIPTION
**Pull request by jugonzal07**
first cut at 23-8 test json

### CODE REVIEW

File: rct229/ruletest_engine/ruletest_jsons/section23/rule_23_8.json [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+                                                "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                                "terminals": [
+                                                    {
+                                                        "id": "VAV Air Terminal 2",
+                                                        "is_supply_ducted": true,
+                                                        "type": "VARIABLE_AIR_VOLUME",
+                                                        "served_by_heating_ventilating_air_conditioning_system": "System 7",
+                                                        "heating_source": "HOT_WATER",
+                                                        "heating_from_loop": "Boiler Loop 1"
+                                                    }
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/section23/rule_23_8.json:109]: I believe we also need ```output_validation_points``` key here as well. Could you double-check this?

**Review by yunjoonjung-PNNL — COMMENTED** [review]
I'd appreciate it if you could check out my comment.

File: rct229/ruletest_engine/ruletest_jsons/section23/rule_23_8.json (hunk 1 above)
Comment (yunjoonjung-PNNL) [reply rct229/ruletest_engine/ruletest_jsons/section23/rule_23_8.json:109]: Juan, I just realized this.

I believe the ```output_validation_points``` should be located under the ```fan_system``` not under the ```fan``` in the ```terminals```. Could you verify and address this?

Comment (jugonzal07) [reply rct229/ruletest_engine/ruletest_jsons/section23/rule_23_8.json:109]: Hi Yun Joon,

Per the schema, the output validation points are at the Fan and not FanSystem level:

https://github.com/pnnl/ruleset-checking-tool/blob/develop/rct229/schema/ASHRAE229.schema.json#L2716-L2721

Comment (yunjoonjung-PNNL) [reply rct229/ruletest_engine/ruletest_jsons/section23/rule_23_8.json:109]: @jugonzal07 

There can be the ```supply_fans``` key under the ```fan_system``` key and ```supply_fans``` key can also have the ```output_validation_points``` key, which is from the [23-8 RDS](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/section23/Rule23-8.md#:~:text=fan_b.output_validation_points) - fan_b is from the ```supply_fans```.

Please let me know if this doesn't make sense!

Comment (yunjoonjung-PNNL) [reply rct229/ruletest_engine/ruletest_jsons/section23/rule_23_8.json:109]: My comment from yesterday was wrong. I should've said that the ```output_validation_points``` should be located under the ```supply_fans``` under the ```fan_system```. My apologies!

Comment (jugonzal07) [reply rct229/ruletest_engine/ruletest_jsons/section23/rule_23_8.json]: Understood! Yes I see what you mean now.

I have pushed an update. You should see the fan_system supply_fans utilizing these output validation points and not the terminal fan.

**Review by yunjoonjung-PNNL — APPROVED** [review]

**Review by weilixu — APPROVED** [review]

**Pull request merged**